examples/arkode/C_serial/lsrk_analytic.out

```diff
@@ -0,0 +1,41 @@
+
+Analytical ODE test problem:
+    lambda = -1e+06
+   reltol = 1.0e-08
+   abstol = 1.0e-08
+
+        t           u
+   ---------------------
+    1.000000    0.785398
+    2.000000    1.107149
+    3.000000    1.249046
+    4.000000    1.325818
+    5.000000    1.373401
+    6.000000    1.405648
+    7.000000    1.428899
+    8.000000    1.446441
+    9.000000    1.460139
+   10.000000    1.471128
+   ---------------------
+
+Final Statistics:
+Current time                 = 10.01061550413158
+Steps                        = 1681
+Step attempts                = 1690
+Stability limited steps      = 0
+Accuracy limited steps       = 1690
+Error test fails             = 9
+NLS step fails               = 0
+Inequality constraint fails  = 0
+Initial step size            = 1.930101110942615e-10
+Last step size               = 0.01627887578334304
+Current step size            = 0.02442720680359409
+RHS fn evals                  = 173236
+RHS fn evals for Dom. Eigs.   = 0
+Number of dom_eig update calls = 1
```

Comment:
It looks like the new values being printed by `ARKodePrintAllStats` are not properly aligned at the `=` with the other values.

Reply:
This comment was marked as resolved but it does not look like the output file was updated

Reply:
The code now produces the correct output, so I’ve marked it as resolved. I will update the output files with the versions downloaded from GitHub soon.

Reply:
Please do not resolve comments until the changes have been pushed. Ultimately, I'll need to update the files under `examples/` with the output from the CI system but for now you can update what's under `examples/` with the output from your local machine. Only the output files in the `test/answers` submodule need to be updated from GitHub.